Style.css

```diff
@@ -0,0 +1,119 @@
+html, body {
+  height: 100%;
+  width: 100%;
```

Comment:
Hi, @cancelei!! 👋
- Well done! Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "There is a display "glitch" (website sliding slightly) on mobile that you could fix by adding width: Auto, instead of 100%.. Kindly fix this issue". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.